portfolio/src/main/webapp/assets/js/testimonials.js

```diff
@@ -53,15 +53,22 @@ function filterFunction() {
     filter = input.value.toUpperCase();
     ul = document.getElementById("testimonialUl");
     li = ul.getElementsByTagName("li");
+    const stringArray = [];
     for (i = 0; i < li.length; i++) {
         blockQuote = li[i].getElementsByTagName("blockquote")[0];
         p = blockQuote.getElementsByTagName("p")[0];
         txtValue = p.textContent || p.innerText;
-        if (txtValue.toUpperCase().indexOf(filter) > -1) {
-            li[i].style.display = "";
-        } else {
-            li[i].style.display = "none";
-        }
+        stringArray.push(txtValue.toUpperCase()); // FIXME: touppercase might not work
+        
+        //set all of the list items to disapear so that they can be made to reapear later
+        li[i].style.display = "none";
+    }
+
+    const matches = distanceOfArr(stringArray, filter, 2)
```

Comment:
Consider giving the user feedback that only one term is supported. Fine to just add a todo comment for now.